core/src/main/resources/lib/hudson/iconSize.jelly

```diff
@@ -37,7 +37,7 @@ THE SOFTWARE.
           </j:when>
           <j:otherwise>
             <li>
-              <a class="yui-button link-button" href="${rootURL}/iconSize?${sz}">
+              <a tooltip="${%Use} ${title.toLowerCase()} ${%icons}" class="jenkins-button jenkins-button--transparent" href="${rootURL}/iconSize?${sz}">
```

Comment:
Should be a parameterized localizable expression to account for different grammars.